README.md

```diff
@@ -230,12 +230,21 @@ TurboConfig.shared.userAgent = "Custom (Turbo Native)"
 
 ### Customize the web view and web view configuration
 
-A block is used because a new instance is needed for each web view.
+A closure is used because a new instance is needed for each web view. The closure has a `WKWebView` argument that's pre-built and ready to be customized.
```

Comment:
```suggestion
A closure is used because a new instance is needed for each web view. The closure has a `WKWebViewConfiguration` argument that's pre-built and ready to be customized and assigned to a new web view.
```